packages/workbox-build/src/lib/cdn-utils.js

```diff
@@ -24,7 +24,10 @@ const getVersionedCDNUrl = () => {
 };
 
 const getModuleUrl = (moduleName, buildType) => {
-  return `${getVersionedCDNUrl()}/${moduleName}.${buildType.slice(0, 4)}.js`;
+  if (buildType) {
```

Comment:
Warning: this is pedantic.

Is there any chance you'd be willing to get the modules package.json and make sure the build type is valid (i.e. if pkg.workbox.prodOnly && buildtype && buildType === 'dev' then throw).

Only reason is that I'd love for this function to be as fool proof as possible just so further down the line we don't get caught out.

The inverse is also possible when it should be if !pkg.workbox.prodOnly && !buildType throw buildType required.